### DISCUSSION TITLE
DIRECT CAMERA IMAGES UPLOAD FAIL

### DISCUSSION THREAD

**Discussion by dev-techguy** [livewire/livewire#3591] [7. Bugs (archived)]
Having some trouble
On laravel livewire upload all files selected from the drive either download do upload successfully. But all camera photos fail kindly advice.

![Screenshot from 2021-08-26 14-06-07](https://user-images.githubusercontent.com/36882600/130958643-9f84d178-9c8c-4849-bd74-25871cda88c8.png)

**Comment by jonneroelofs** [comment]
Looks like you are getting a validation error. Maybe the image is larger than the allowed size for your application?
